debian/control

```diff
@@ -42,7 +42,7 @@ Description: Open Lighting Architecture - Python Classes
 Package: ola-rdm-tests
 Section: libs
 Architecture: all
-Depends: ola (>= ${source:Version}), ola (<< ${source:Version}.~), ola-python (= ${binary:Version}), ${python:Depends}, ${misc:Depends}, python-numpy, libjs-jquery, libjs-jquery-ui, lsb-base
+Depends: ola (>= ${source:Version}), ola (<< ${source:Version}.~), ola-python (= ${source:Version}), ${python3:Depends}, ${misc:Depends}, python3-numpy, libjs-jquery, libjs-jquery-ui, lsb-base
```

Comment:
I'm no expert, but I'm surprised this would need changing from binary to source.

Reply:
I don't exactly remember why I did this so I can try without.

Reply:
I guess is it also strange that we require a source version of `ola` here?

Reply:
Pushed a change, if the test steps run fine then it works fine!

Reply:
Seems fine!

Reply:
The source version for the other one went in here:
https://salsa.debian.org/wouter/ola/-/commit/dc0081bce482e87f1413ab0b2ad52876788a0360

I'd trust @yoe 's Debian Fu over my random guessing! But it feels like it could make sense. The ola package is a source package where as ola-python is binary only I believe...

Reply:
Yeah assuming the ola-python is binary then it makes sense for ola-rdm-tests to depend on ola-python binary. I think my reverted change makes sense.